### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by SiirRandall** [debauchee/barrier#1989]
### What happened?

I haven't seen any updates for two years. I love the hard work that went into this. I want to make sure this project isn't dead, and updates will be made if needed. 

### Version

v2.0.0-RC1

### What OSes are you seeing the problem on? (Check all that apply)

Linux, Windows, macOS

### What OS versions are you using?

Fedora 40, macOS 14, WIndows 11 Pro

**Comment by m-graf** [comment]
Open a PR, and contribute my friend. This project is not dead. :)

**Comment by stefnotch** [comment]
This project has moved to https://github.com/input-leap/input-leap

However, as commented below

> It's worth noting that while input-leap is more active, they have yet to release a build that they recommend using. They still recommend using Barrier 2.4.0 until they finally release something stable.
> https://github.com/debauchee/barrier/issues/1989#issuecomment-1818771399

**Comment by m-graf** [comment]
> This project has moved to https://github.com/input-leap/input-leap

This is not entirely true. The two most active maintainers have forked this project, and have started the project in the link mentioned. I point the out because the discussion (https://github.com/input-leap/input-leap/issues/1414) was obviously heated, and was muted. 

edit: just found the link to the open discussion: https://github.com/input-leap/input-leap/discussions/1427

neat :)

**Comment by stefnotch** [comment]
Well yes, of course. The project has been forked, and the currently most promising looking fork is input-leap. And the owner of the barrier repository is at the moment not involved in input-leap. 

Anyways, I chose to communicate it that way ("This project has moved to ..."), since I just wanted to give a simplified answer to anyone who's just looking for a solution.

The discussion thread is interesting. I'm glad to see that input-leap seems to be in good hands :)

**Comment by wccrawford** [comment]
It's worth noting that while input-leap is more active, they have yet to release a build that they recommend using.  They still recommend using Barrier 2.4.0 until they finally release something stable.

**Comment by sgon00** [comment]
Yeah, it's dead. Installation is broken (no openssl key created). No wayland support. Not usable at all.

**Comment by yllekz** [comment]
This is the first time I'm hearing that the project has been forked. I think there should definitely be a much more forthcoming communication to users of this project...

**Comment by diegorodriguezv** [comment]
> Open a PR, and contribute my friend. This project is not dead. :)

There are a lot of PR's that are more than 3 years old. And almost a thousand open issues. I know that in the open source world instead of asking others to fix problems you should fix them yourself and contribute, but as a developer I don't want to waste my time creating a PR that won't get merged.
I would think twice before contributing here.

**Comment by dog6o** [comment]
>Is this project dead?
Exactly

Especially after upgrading macOS to Ventura, Barrier's performance has been terrible, with the mouse cursor getting stuck and stuttering.
input-leap is a sub-alpha project that doesn't even have binaries, so there are no expectations and no future.
We would like to present an alternative to users who are currently in despair over Barrier's lack of support for the latest OS and its unresolved problems.
The best way to do this is to use a free binary version of Synergy available somewhere.
If that's the case, then Barrier's fatal bugs have almost been resolved, so I recommend it because you don't have to wait impatiently for input-leap, which might release a binary in 10 years.

**Comment by dog6o** [comment]
>Is this project dead?

exactly.

Especially after upgrading macOS to Ventura, Barrier's performance has been terrible, with the mouse cursor getting stuck and stuttering.
input-leap is a sub-alpha project that doesn't even have binaries, so there are no expectations and no future.
We would like to present an alternative to users who are currently in despair over Barrier's lack of support for the latest OS and its unresolved problems.
The best way to do this is to use a free binary version of Synergy available somewhere.
If that's the case, then Barrier's fatal bugs have almost been resolved, so I recommend it because you don't have to wait impatiently for input-leap, which might release a binary in 10 years.

**Comment by myrandor** [comment]
The last commit of this project is more than 2 years ago, and there's a lot of PR to be tested and approved.

Is @walker0643 or @p12tic still involve in this project? I'd love to contribute to help making it more stable and active! 😊

**Comment by stefnotch** [comment]
@myrandor I would contact the developers of input leap https://github.com/input-leap/input-leap?tab=readme-ov-file#input-leap

**Comment by nbolton** [comment]
Barrier postmortem: [Why did Barrier fail?](https://symless.com/synergy/news/why-did-barrier-fail)

**Comment by walker0643** [comment]
@myrandor It would be great to get the project caught up with the PRs. It's been quite a while since we've had an active developer, so there is definitely work to be done. Jump on the Discord server If you want to discuss: https://discord.gg/GbEud2RVV4 - Thanks!

**Comment by yllekz** [comment]
If you want a 1:1 app replacement, I jumped to input-leap from this and things continued as normal. I think the two apps are even capable of talking to one another.

**Comment by nbolton** [comment]
Please try Deskflow or Input Leap as Barrier is no longer in development.

https://github.com/deskflow/deskflow
https://github.com/input-leap/input-leap

The 'Insights' tab of each GitHub project may help you decide which one to use.
Both are compatible with Barrier client/server so you can try on one of your computers.

If this is still an issue in those projects, we would appreciate a cross-post of this issue.

**Comment by sithlord48** [comment]
@walker0643  Why not just work with us on Deskflow ? There is nothing here but tons of bugs and old code to update.. Most of that is now done in Deskflow. Instead of backporting and fixing here lets work together to make better software for everyone.
